### PR TITLE
FIXED #110 Scrolling Effect Added in Session Detail Activity

### LINKED ISSUE #110: UI BUG: Scrolling Effect Missing in Session Detail Activity
bug_op

### PR DESCRIPTION
**Pull request by opticod**
#110 Added Scroll View in SessionDetail Activity

optimised

### CODE REVIEW

**Comment by batraabhishek** [comment]
@creativepsyco Is this good to merge? Build is failing though.

**Comment by mananwason** [comment]
@batraabhishek We need to restart the build if it fails. Otherwise the code looks fine.

**Comment by opticod** [comment]
@batraabhishek
Its the Travis Inconsistency ...
I have one earlier incident...It had failed first time and then it passed..

2c3275ae-9bb9-11e5-9b9c-bbce7e2fa153
a5c001a2-9bb9-11e5-9ea8-835d3624afc5

And with the code...i have only modified the layout file and attaching a ScrollView.

File: app/src/main/res/layout/sessions_detail.xml [hunk 1]
```diff
@@ -11,108 +11,111 @@
...
 
-            <LinearLayout
+            <android.support.v7.widget.CardView
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:orientation="vertical">
+                android:layout_marginBottom="@dimen/card_margin"
+                android:layout_marginLeft="@dimen/card_margin"
+                android:layout_marginRight="@dimen/card_margin"
+                android:layout_marginTop="24dp">
```
Comment (mananwason) [inline app/src/main/res/layout/sessions_detail.xml]: @opticod please try to create a style in styles.xml and then use the style here.

**Comment by mananwason** [comment]
@opticod Restart it.

**Comment by opticod** [comment]
@mananwason I have no permissions to restart owner's build...I can only restart forked repos

File: app/src/main/res/layout/sessions_detail.xml (hunk 1 above)
Comment (batraabhishek) [inline app/src/main/res/layout/sessions_detail.xml]: @mananwason I am going to work on it. Will be doing some UI changes :). @opticod No need to create a style. I will be editing it any ways. :)

**Comment by opticod** [comment]
@mananwason I did

**Comment by mananwason** [comment]
Yeah you have to restart that only since only this PR is failing.

**Comment by creativepsyco** [comment]
you ust rebase with current master first.

**Comment by opticod** [comment]
@creativepsyco @mananwason @batraabhishek Thanks God.. Finally it has passed .. :smiley:

**Comment by opticod** [comment]
@creativepsyco I have rebased it.Please review it.Thank You.

**Comment by creativepsyco** [comment]
looks fine @opticod ! :+1: ask one of your mentors, either @batraabhishek or @mananwason to merge this.

**Comment by opticod** [comment]
@batraabhishek @mananwason Please merge this.

**Comment by mananwason** [comment]
@opticod done :smile:

**Comment by opticod** [comment]
:smile:

**Pull request merged**